packages/app/src/debug/GroupedDebugFailedTest.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div
+    v-for="(group, j) in props.groups"
+    :key="j"
+    class="flex flex-col flex-start w-full pl-16px justify-center grouped-row-class hover:bg-gray-50"
+    data-cy="grouped-row"
+  >
+    <div
+      class="flex flex-start flex-row font-normal border-b-gray-100 h-12 text-base text-gray-700 gap-x-2.5 items-center non-italic"
+      data-cy="test-failed-metadata"
+    >
+      <StatsMetadata
+        :order="['GROUP_NAME', 'OS', 'BROWSER']"
+        :groups="[group]"
+        :group-name="group.groupName!"
+      />
+      <div
+        data-cy="debug-artifacts"
+        class="flex flex-grow space-x-4.5 opacity-0 pr-18px justify-end grouped-row-artifacts"
```

Comment:
```suggestion
        class="flex flex-grow space-x-4.5 opacity-0 px-18px justify-end grouped-row-artifacts"
```
Make sure there is also some left padding to keep the artifact links away from the stats metadata